tests/core/test_serialization.py

```diff
@@ -20,7 +20,7 @@
 from nemo.core.classes.common import Serialization
 
 
-class SerializationTest:
+class TestSerialization:
```

Comment:
Good catch! Those tests were skipped before, right?

Reply:
Yep, I tried running it locally and it skipped all of them 😓

Reply:
I have covered more cases in EFF:
https://gitlab-master.nvidia.com/tlt/eff/-/blob/main/tests/test_nemo_convert.py

@titu1994 You can simply grab them from there if you want 👍